runtime/doc/ui.txt

```diff
@@ -219,8 +219,9 @@ the editor.
 
 ["busy_start"]
 ["busy_stop"]
-	Nvim started or stopped being busy, and possibly not responsive to
-	user input. This could be indicated to the user by hiding the cursor.
+	This event is misnamed and does not actually have anything to do with
```

Comment:
I would reverse the order of these sentences (start with telling the reader what to actually do :)